e2e/portalicious/pages/LoginPage.ts

```diff
@@ -30,6 +33,25 @@ class LoginPage {
       url.pathname.startsWith('/en/projects'),
     );
   }
+
+  async loginTest(username?: string, password?: string) {
```

Comment:
This method should be unnecessary. You should be able to use `login()` with `skipUrlCheck = true`

Reply:
Done